code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -28,12 +28,20 @@
 /obj/structure/closet/crate/can_close()
 	return TRUE
 
-/obj/structure/closet/crate/open()
+/obj/structure/closet/crate/open(irregular_open = FALSE)
```

Comment:
I'm not 100% on the param name. `irregular_open` doesn't tell much about what it does when it's `TRUE` or `FALSE`. Adding comments will help with that. Though a more descriptive param name would be better. (combination is also always good)

Reply:
It's now by_hand and confirms that it was opened by hand, rather than if it was opened improperly